frontend/src/features/SideWindow/MissionForm/MainForm/index.tsx

```diff
@@ -81,29 +94,31 @@ export function MainForm({ onTypeChange }: MainFormProps) {
         <CustomFormBody>
           <FormikDateRangePicker
             baseContainer={newWindowContainerRef.current}
+            isCompact
+            isStringDate
             label="Début et fin de mission"
-            name="dateRange"
+            // `inputStartDateTimeUtc` & `inputEndDateTimeUtc` in API
+            name="inputDateTimeRangeUtc"
```

Comment:
inputDateTimeRangeUtc should be named `dateTimeRangeUtc`

Reply:
Yes I agree but I was trying to follow the API despite this strange naming... I will update it.